packages/block-editor/src/components/inspector-controls/index.native.js

```diff
@@ -19,7 +18,13 @@ const { Fill, Slot } = createSlotFill( 'InspectorControls' );
 const FillWithSettingsButton = ( { children, ...props } ) => {
 	return (
 		<>
-			<Fill { ...props }>{ children }</Fill>
+			<Fill { ...props }>
+				{
+					<BottomSheetConsumer>
+						{ () => <View>{ children }</View> }
```

Comment:
Can we change the view into the fragment?

Reply:
Actually we cannot, since `View` has to be a parent and it's necessary to avoid a crash during `LayoutAnimation` on Android.